test/index_spec.js

```diff
@@ -254,5 +254,19 @@ describe('textPosition', () => {
         toRange(root, {start, end})
       }, 'End offset of position selector is out of range')
     })
+
+    it('handles an empty root element', () => {
+      let root = document.createElement('div');
+      assert.throws(() => {
+        toRange(root, {start:0, end: 0})
+      }, 'Start offset of position selector is out of range');
+    })
+
+    it('handles a root element with an empty text node', () => {
+      let root = document.createElement('div');
+      root.appendChild(document.createTextNode())
```

Comment:
`document.createTextNode()` needs an empty string argument. At least under Chromium, locally, it seems to throw without it.